lib/rubocop/cop/rspec/empty_example_group.rb

```diff
@@ -88,8 +93,12 @@ class EmptyExampleGroup < Base
         #   @return [Array<RuboCop::AST::Node>] matching nodes
         def_node_matcher :example_or_group_or_include?, <<~PATTERN
           {
+            #{Examples::ALL.send_pattern}
+            #{Examples::SKIPPED.send_pattern}
```

Comment:
Why SKIPPED and PENDING do not match the ALL pattern?

Reply:
Good catch, thank you. Don't know what I was thinking.